ConfigMappingValidatorTest failing when deploying snapshots

@radcortez the build deploying the snapshots failed with the following error tonight. Could you have a look?

Failing build is here if you need details: https://github.com/quarkusio/quarkus/runs/2606481261?check_suite_focus=true .

```
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.946 s <<< FAILURE! - in io.quarkus.hibernate.validator.test.config.ConfigMappingValidatorTest
Error:  io.quarkus.hibernate.validator.test.config.ConfigMappingValidatorTest  Time elapsed: 0.946 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:544)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$8(ClassBasedTestDescriptor.java:368)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:368)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:192)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:136)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:637)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.runner.bootstrap.StartupActionImpl.run(StartupActionImpl.java:181)
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:500)
	... 40 more
Caused by: javax.enterprise.inject.spi.DeploymentException: Configuration validation failed:
	server.username does not map to any root
	server.password does not map to any root
	at io.quarkus.arc.runtime.ConfigRecorder.registerConfigMappings(ConfigRecorder.java:52)
	at io.quarkus.deployment.steps.ConfigBuildStep$registerConfigClasses-2116437784.deploy_0(ConfigBuildStep$registerConfigClasses-2116437784.zig:132)
	at io.quarkus.deployment.steps.ConfigBuildStep$registerConfigClasses-2116437784.deploy(ConfigBuildStep$registerConfigClasses-2116437784.zig:40)
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:502)
	... 47 more
Caused by: io.smallrye.config.ConfigValidationException: Configuration validation failed:
	server.username does not map to any root
	server.password does not map to any root
	at io.smallrye.config.ConfigMappingProvider.mapConfiguration(ConfigMappingProvider.java:818)
	at io.smallrye.config.ConfigMappingProvider.mapConfiguration(ConfigMappingProvider.java:777)
	at io.smallrye.config.ConfigMappings.mapConfiguration(ConfigMappings.java:54)
	at io.smallrye.config.ConfigMappings.registerConfigMappings(ConfigMappings.java:36)
	at io.quarkus.arc.runtime.ConfigRecorder.registerConfigMappings(ConfigRecorder.java:50)
	... 50 more

[INFO] Running io.quarkus.hibernate.validator.test.ConstraintOnStaticMethodTest
2021-05-18 02:48:03,742 WARN  [io.qua.hib.val.deployment] (build-7) Hibernate Validator does not support constraints on static methods yet. Constraints on io.quarkus.hibernate.validator.test.ConstraintOnStaticMethodTest$MyUtil#void validateName(@Pattern(regexp = "A.*") java.lang.String name) are ignored.
2021-05-18 02:48:04,388 INFO  [io.quarkus] (main) Quarkus 999-SNAPSHOT on JVM started in 1.429s. Listening on: http://localhost:8081
2021-05-18 02:48:04,388 INFO  [io.quarkus] (main) Profile test activated. 
2021-05-18 02:48:04,388 INFO  [io.quarkus] (main) Installed features: [cdi, hibernate-validator, resteasy-jsonb, smallrye-context-propagation]
2021-05-18 02:48:04,458 INFO  [io.quarkus] (main) Quarkus stopped in 0.023s
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.069 s - in io.quarkus.hibernate.validator.test.ConstraintOnStaticMethodTest
```

--- a/extensions/hibernate-validator/deployment/src/test/java/io/quarkus/hibernate/validator/test/config/ConfigMappingValidatorTest.java
+++ b/extensions/hibernate-validator/deployment/src/test/java/io/quarkus/hibernate/validator/test/config/ConfigMappingValidatorTest.java
@@ -21,7 +21,7 @@ public class ConfigMappingValidatorTest {
     @RegisterExtension
     static final QuarkusUnitTest UNIT_TEST = new QuarkusUnitTest().setArchiveProducer(
             () -> ShrinkWrap.create(JavaArchive.class)
-                    .addAsResource(new StringAsset("server.host=localhost\n"), "application.properties"));
+                    .addAsResource(new StringAsset("validator.server.host=localhost\n"), "application.properties"));
 
     @Inject
     Config config;
@@ -30,10 +30,10 @@ public class ConfigMappingValidatorTest {
     void validator() {
         assertThrows(ConfigValidationException.class,
                 () -> config.unwrap(SmallRyeConfig.class).getConfigMapping(Server.class),
-                "server.host must be less than or equal to 3");
+                "validator.server.host must be less than or equal to 3");
     }
 
-    @ConfigMapping(prefix = "server")
+    @ConfigMapping(prefix = "validator.server")
     public interface Server {
         @Max(3)
         String host();

